pkgs/tools/networking/wait-for-it/default.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "wait-for-it";
+
+  src = fetchFromGitHub {
+    owner = "vishnubob";
+    repo = "wait-for-it";
+    rev = "c096cface5fbd9f2d6b037391dfecae6fde1362e";
+    sha256 = "1k1il4bk8l2jmfrrcklznc8nbm69qrjgxm20l02k01vhv2m2jc85";
+  };
+  installPhase = ''
+    mkdir -p $out/bin
+    cp ./wait-for-it.sh $out/bin
+  '';
```

When overriding `installPhase`, you should also provide `runHook preInstall` and `runHook postInstall`.

It is also nicer to just use `install` command.

```suggestion
  installPhase = ''
    runHook preInstall

    install -Dt $out/bin ./wait-for-it.sh

    runHook postInstall
  '';
```

---

@jtojnar : done. Out of curiosity, I could find virtually no documentation on the `install` function/procedure here: https://nixos.org/nixpkgs/manual/

What does it do? Any links would be super helpful.

---

It is just a standard UNIX utility, see [`install(1)`](https://linux.die.net/man/1/install).